index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&family=Playfair+Display&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section id="header">
+      
+        
+<div class="Menu">   
+<div class="Menu-line"></div>
+<div class="Menu-line"></div>
+<div class="Menu-line"></div>
+</div>   
+
+<img src="images/Shape (1).png" alt="" class="image1">
+<img src="images/Shape (2).png" alt="" class="image2">
+
+    </section>
+
+    <section id="headline">
+
+        <div class="Intro">
+        <h1>Ricardo Egas</h1>
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.</p>
+        <p>If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+
+        <button class="Collaboration">Start Collaboration</button>
+
+        <img src="images/Screenshot 2023-03-23 131136.png" alt="" class="image3">
+
+        </div>
+     
+
+        
+
+    </section>
```

Comment:
Nice job building the headline section😉 let's make it look better by sticking to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=sEK7CqFA4CCNzxBm-0)😉 remember we are required to stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=sEK7CqFA4CCNzxBm-0) in this milestone and it is a good display of professionalism when a developer can reproduce a design without making alterations😉 Kindly see the image below, notice the differences and make the necessary changes😉

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="300" alt="your headline" src="https://user-images.githubusercontent.com/64643995/227253432-a12d5c07-833d-4421-bb1e-db69cee33027.png" width="300">|<img width="300" alt="design" src="https://user-images.githubusercontent.com/64643995/227253600-e1d941f0-1197-42ad-8d41-58ce78721a6c.png" width="300">|


Kindly take note of the `spacings`, `font-properties` and make the necessary changes😉